reducers/storage.ts

```diff
@@ -110,6 +111,47 @@ function storage(state: Record<string, any> = {}, action: GenericAction) {
     }
 }
 
+function migrateDrafts(state: any) {
+    const drafts: any = {};
+    for (const storageKey in state) {
+        if (storageKey.startsWith('draft')) {
```

Comment:
Nitpicking, but we've gotten in the habit of exiting early from things to avoid nesting stuff as deeply, so this could be written like
```
for (loop) {
    if (!storageKey.startsWith('draft')) {
        continue;
    }
    
    if (storageDraft.value?.channelId) {
        // No migration is needed
        continue;
    }
    
    drafts[storageKey] = /* TODO */;
}
```